spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkTable.java

```diff
@@ -313,4 +405,29 @@ private static CaseInsensitiveStringMap addSnapshotId(CaseInsensitiveStringMap o
 
     return options;
   }
+
+  private static Object[] transform(StructType schema, Object[] values) {
+    Object[] resultValues = new Object[values.length];
```

Comment:
I don't think we need this, I believe we should be able to just use the rows as they come out of the Partition metadata table

Reply:
deleted this code